packages/components/src/call-out/index.tsx

```diff
@@ -0,0 +1,67 @@
+import {
+	Card,
+	__experimentalVStack as VStack,
+	__experimentalHStack as HStack,
+	__experimentalHeading as Heading,
+	Icon,
+} from '@wordpress/components';
+import clsx from 'clsx';
+import { forwardRef } from 'react';
+import type { CallOutProps } from './types';
+import './styles.scss';
+
+function UnforwardedCallOut(
+	{ title, icon, illustration, actions, children }: CallOutProps,
+	ref: React.ForwardedRef< HTMLElement >
+) {
+	return (
+		<Card ref={ ref } className="call-out">
+			<HStack
+				ref={ ref }
+				className="call-out-h-container"
+				spacing="6"
+				alignment="stretch"
+				justify="stretch"
+				expanded={ false }
+			>
+				<VStack justify="flex-start" alignment="flex-start" spacing="4">
+					{ icon && <Icon icon={ icon } /> }
+					<Heading level="6" className="call-out-title">
+						{ title }
+					</Heading>
+					{ children }
+					{ actions }
+				</VStack>
+				{ illustration && (
+					<VStack justify="stretch" alignment="stretch" className="call-out-illustration-container">
+						{ illustration }
+					</VStack>
+				) }
+			</HStack>
+		</Card>
+	);
+}
+
+function UnforwardedCallOutIllustration(
+	{ className, alt, ...rest }: React.ImgHTMLAttributes< HTMLImageElement >,
+	ref: React.ForwardedRef< HTMLImageElement >
+) {
+	return (
+		<img
+			ref={ ref }
+			className={ clsx( 'call-out-illustration', className ) }
+			alt={ alt } // Forwarded explicitly to silence warnings
+			{ ...rest }
+		/>
+	);
+}
+
+/**
+ * The CallOut component is a visually prominent layout element used to
+ * highlight a key feature, message, or promotional content. It typically
+ * includes a title, supporting description, illustration and/or icon,
+ * and a single call-to-action (CTA) button or link.
+ */
+export const CallOut = forwardRef( UnforwardedCallOut );
```

Comment:
`Breadcrumbs` and `SummaryButton` also forward the ref, so I thought I should too.

Although is it wise to do this when we haven't got a use for it? I'm simply forwarding it to the parent element, how do I know if that's the best element the consumer would want a ref to?

Reply:
> `Breadcrumbs` and `SummaryButton` also forward the ref, so I thought I should too.
> 
> Although is it wise to do this when we haven't got a use for it? I'm simply forwarding it to the parent element, how do I know if that's the best element the consumer would want a ref to?

I feel like we should have a consistent stance on this, to avoid confusion both for component developers and component consumers. My instinct is that we could always accept the `ref` and always assign it to the top-most wrapping element, but maybe that misses some use-cases I'm not thinking about?

Reply:
Without know the use case I think the most wrapping element is the most obvious candidate. It's most likely to be useful for something like collision detection.

Reply:
Noting that [forwardRef will be deprecated](https://react.dev/blog/2024/12/05/react-19#improvements-in-react-19) in the near future.

Reply:
As far as I understand, `forwardRef` is going away, but only because components receive the `ref` by default. But that doesn't impact what's being discussed here, which is if/what to do with that `ref`.